src/components/Images/BlankImage.tsx

```diff
@@ -0,0 +1,175 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+interface Props {
+  title?: string
+  size?: 's' | 'm' | 'l'
```

Comment:
[MUST] To accept style changes from the user side, expose className props as optional and pass it to the `Wrapper`.